sn_client/src/api.rs

```diff
@@ -67,7 +67,7 @@ impl Client {
     ///
     /// Optionally specify the duration for the connection timeout.
     ///
-    /// Defaults to 180 seconds.
+    /// Defaults to `CONNECTION_TIMEOUT`.
```

Comment:
`CONNECTION_TIMEOUT` isn't a `pub` value, so this has no meaning in public API docs. Perhaps make it a `pub const` and write `[CONNECTION_TIMEOUT]` so it's hyperlinked.